erp/forms.py

```diff
@@ -607,6 +607,8 @@ class Meta:
     published = forms.BooleanField(
         label="Je souhaite mettre en ligne cette fiche d'établissement immédiatement",
         required=False,
+        initial=True,
+        widget=forms.CheckboxInput(attrs={"checked": True}),
```

Comment:
Why is this necessary? Also, are you sure it's not `"checked": "checked"`, as in HTML?

Reply:
Same result. Inital=True is not sufficient, and doesn't check the box, even if it could.